tests/libcxx/expected_results.txt

```diff
@@ -891,25 +843,296 @@ std/iterators/iterator.primitives/iterator.operations/advance.pass.cpp SKIPPED
+std/algorithms/alg.sorting/alg.set.operations/set.symmetric.difference/ranges_set_symmetric_difference.pass.cpp FAIL
+std/algorithms/alg.sorting/alg.set.operations/set.union/ranges_set_union.pass.cpp FAIL
+std/algorithms/alg.sorting/alg.sort/is.sorted/ranges.is_sorted.pass.cpp:0 FAIL
+std/algorithms/alg.sorting/alg.sort/is.sorted/ranges.is_sorted_until.pass.cpp:0 FAIL
+std/algorithms/alg.sorting/alg.sort/partial.sort.copy/ranges_partial_sort_copy.pass.cpp:1 FAIL
+std/algorithms/alg.sorting/alg.sort/stable.sort/ranges.stable.sort.pass.cpp:1 FAIL
+std/algorithms/algorithms.results/in_found_result.pass.cpp:0 FAIL
+std/algorithms/algorithms.results/min_max_result.pass.cpp:0 FAIL
+std/algorithms/ranges_result_alias_declarations.compile.pass.cpp FAIL
+std/algorithms/ranges_robust_against_dangling.pass.cpp FAIL
+std/algorithms/ranges_robust_against_differing_projections.pass.cpp FAIL
+std/algorithms/ranges_robust_against_nonbool_predicates.pass.cpp FAIL
+std/algorithms/ranges_robust_against_omitting_invoke.pass.cpp FAIL
+std/algorithms/ranges_robust_against_proxy_iterators.pass.cpp FAIL
+std/algorithms/robust_against_proxy_iterators_lifetime_bugs.pass.cpp FAIL
+std/concepts/concepts.lang/concept.default.init/default_initializable.compile.pass.cpp:0 FAIL
+std/containers/associative/map/map.access/iterator.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/construct_iter_iter.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/construct_iter_iter_alloc.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/construct_size.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/construct_size_value.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/construct_size_value_alloc.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/emplace_back.pass.cpp FAIL
+std/containers/sequences/vector.bool/enabled_hash.pass.cpp FAIL
+std/containers/sequences/vector.bool/move.pass.cpp:0 FAIL
+std/containers/sequences/vector.bool/vector_bool.pass.cpp FAIL
+std/containers/sequences/vector/iterators.pass.cpp:0 FAIL
+std/containers/sequences/vector/reverse_iterators.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.cons/construct_iter_iter.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.cons/construct_iter_iter_alloc.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.cons/construct_size.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.cons/construct_size_value.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.cons/construct_size_value_alloc.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.erasure/erase.pass.cpp:0 FAIL
+std/containers/sequences/vector/vector.erasure/erase_if.pass.cpp:0 FAIL
+std/input.output/filesystems/fs.op.funcs/fs.op.remove_all/toctou.pass.cpp FAIL
+std/input.output/iostream.format/quoted.manip/quoted_traits.compile.pass.cpp FAIL
+std/iterators/iterator.primitives/range.iter.ops/range.iter.ops.distance/iterator_sentinel.pass.cpp FAIL

```

Comment:
No need to file a bug, I've submitted a fix in https://github.com/microsoft/STL/pull/2987. (After which the test fails because of a warning about implicitly converting `ptrdiff_t` to `int`, which I've fixed upstream.)